<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameView</class>
 <widget class="QMainWindow" name="GameView">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">BlueButton[enabled='false']{color:red;}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="whiteCount">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="blackCount">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="whiteImage">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>白方头像</string>
    </property>
   </widget>
   <widget class="QLabel" name="whiteUser">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>白方用户名</string>
    </property>
   </widget>
   <widget class="QLabel" name="blackUser">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>90</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>黑方用户名</string>
    </property>
   </widget>
   <widget class="QLabel" name="blackImage">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>110</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>黑方头像</string>
    </property>
   </widget>
   <widget class="BlueButton" name="whiteRegret">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>93</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>悔棋(剩余)</string>
    </property>
   </widget>
   <widget class="BlueButton" name="blackRegret">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>340</y>
      <width>93</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>悔棋(剩余)</string>
    </property>
   </widget>
   <widget class="BlueButton" name="whiteDeposit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>93</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>托管</string>
    </property>
   </widget>
   <widget class="BlueButton" name="blackGiveup">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>400</y>
      <width>93</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>投降</string>
    </property>
   </widget>
   <widget class="QLabel" name="gameMode">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>游戏模式</string>
    </property>
   </widget>
   <widget class="BlueButton" name="whiteGiveup">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>93</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>投降</string>
    </property>
   </widget>
   <widget class="BlueButton" name="blackDeposit">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>460</y>
      <width>93</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>托管</string>
    </property>
   </widget>
   <widget class="QSlider" name="recallSlider">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>530</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="recallLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>530</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>回忆速度</string>
    </property>
   </widget>
   <widget class="QPushButton" name="voiceCtrl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="voiceSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>21</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:grey;</string>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="voiceNum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QLabel" name="whiteChess">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blackChess">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>90</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>807</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actioReturn"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>:/images/saveIcon.png</normaloff>:/images/saveIcon.png</iconset>
   </property>
   <property name="text">
    <string>保存进度</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset>
     <normaloff>:/images/readIcon.png</normaloff>:/images/readIcon.png</iconset>
   </property>
   <property name="text">
    <string>读取进度</string>
   </property>
   <property name="toolTip">
    <string>读取进度</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset>
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actioReturn">
   <property name="text">
    <string>回到开始界面</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlueButton</class>
   <extends>QPushButton</extends>
   <header>bluebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
